R/infrastructure.R

```diff
@@ -121,6 +142,11 @@ use_rcpp <- function(pkg = ".") {
   ignore_path <- file.path(pkg$path, "src", ".gitignore")
   union_write(ignore_path, c("*.o", "*.so", "*.dll"))
 
+  if (use_inst) {
+    use_directory("inst/include/", pkg = pkg)
+    use_template("inst_include_template", "/src/Makevars")
```

Comment:
FWIW the 'overwrite?' dialog for `use_template` is kind of awkward here (you either need to accept that your original Makevars is overwritten, or you get an error when you cancel / reject the dialog)